frappe/www/printview.py

```diff
@@ -338,6 +338,9 @@ def has_value(df, doc):
 		return False
 
 	elif isinstance(value, string_types) and not strip_html(value).strip():
+		if is_html(value) and "<img" in value:
+			return True
```

Comment:
This is too specific. Just check if fieldtype is TextEditor.

Reply:
What if other fieldtype like data, small text has image value?

Reply:
Fixed!